assets/js/modules/analytics-4/components/widgets/TopCategoriesWidget.js

```diff
@@ -107,10 +107,12 @@ function TopCategoriesWidget( { Widget } ) {
 			field: 'dimensionValues',
 			Component: ( { fieldValue } ) => {
 				const [ categories ] = fieldValue;
-				let categoriesList = JSON.parse( categories.value );
-				if ( ! Array.isArray( categoriesList ) ) {
-					categoriesList = [];
-				}
+
+				const categoriesList =
+					typeof categories?.value === 'string'
+						? categories?.value.split( '; ' )
```

Comment:
This one-character detail got missed in the last update - `categories` will always exist here, because if it's `undefined` it will never be of type `string`. I'll apply the change myself to save the trip!

```suggestion
						? categories.value.split( '; ' )
```